packages/abi/scripts/publish.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+yarn generate:abi
```

Comment:
Do you think this should be here, even if it fails it will continue? Same as above basically